openmetrics/datadog_checks/openmetrics/data/conf.yaml.example

```diff
@@ -0,0 +1,86 @@
+init_config:
+  # Leave empty
+
+instances:
+  # The prometheus endpoint to query
```

Comment:
We should follow the new format with `## @param ...` now. You can refer to #2132 for some examples.